File: apps/biobank_api/src/biobank_api/domain/enums.py
# Kept at the domain root (not under domain/models) so domain.cleaning can import it without
# creating a models-package import cycle.
from __future__ import annotations

from enum import Enum


class Sex(Enum):
    MALE = "male"
    FEMALE = "female"


class Retrieved(Enum):
    # operational = taken during a surgical/clinical procedure; unknown = context not recorded.
    OPERATIONAL = "operational"
    UNKNOWN = "unknown"

File: src/BiobankApi/BiobankApi.Domain/Enums.cs
namespace BiobankApi.Domain;

/// <summary>Biological sex of a patient.</summary>
public enum Sex
{
    Male,
    Female,
}

/// <summary>
/// Whether a sample was retrieved during a clinical/surgical procedure (<c>Operational</c>) or its
/// context was not recorded (<c>Unknown</c>).
/// </summary>
public enum Retrieved
{
    Operational,
    Unknown,
}
